src/components/EventList.js

```diff
@@ -71,7 +76,20 @@ export const EventList = ({
       refreshControl={refreshControl}
       renderItem={renderItem}
       renderSectionHeader={({ section: { title } }) => (
-        <SectionHeader title={momentFormat(title, 'DD.MM.YYYY dddd')} />
+        <SectionHeader
+          title={momentFormat(title, 'DD.MM.YYYY dddd')}
+          onPress={() =>
+            navigation.push(ScreenName.Index, {
+              title: texts.homeTitles.events,
+              query,
+              queryVariables: {
+                ...queryVariables,
+                dateRange: [momentFormat(title, 'YYYY-MM-DD'), momentFormat(title, 'YYYY-MM-DD')]
+              },
+              rootRouteName: ROOT_ROUTE_NAMES.EVENT_RECORDS
+            })
+          }
```

Comment:
I'm getting an error here for Volunteer. I think a redirect needs to be added for the `ScreenName.VolunteerIndex` screen.

![IMG_2044](https://user-images.githubusercontent.com/11755668/214009654-c7fce219-4b15-4fbe-ad2f-58e010bf024f.PNG)

Reply:
there is a weird error on volunteer calendar sections, thanks!
but it is not located in the `EventList`, as tis component is not used for volunteer. i need to find the right place.

Reply:
one problem was the pressable header for volunteer lists, which i have reverted and restricted to default events: 1c1e9e1

Reply:
found the bad spot: 96a259c
changed condition on `query` instead of `contentContainerId`, which can be empty for volunteer as well